rmf_fleet_adapter/src/robot_state_aggregator/main.cpp

```diff
@@ -116,13 +175,4 @@ class RobotStateAggregator : public rclcpp::Node
 
 };
 
-int main(int argc, char* argv[])
```

Comment:
Knit: Since there is no `main()` in this file, I recommend renaming it to `RobotStateAggregator.cpp` to prevent any confusion and help with overall readability.

Reply:
I was trying to initially maintain minimal changes to this file but I agree much better this way.